spec/mailers/candidate_mailer_spec.rb

```diff
@@ -242,4 +242,93 @@
       'course start' => 'September 2021',
     )
   end
+
+  describe '.new_interview' do
+    let(:email) { mailer.new_interview(@application_choice, @interview) }
+
+    before do
+      @interview = create(:interview,
+                          provider: create(:provider, name: 'Hogwards'),
+                          date_and_time: Time.zone.local(2021, 1, 15, 9, 30),
+                          location: 'Hogwarts Castle',
+                          additional_details: 'Bring your magic wand for the spells test')
+      @application_choice = @interview.application_choice
+      build_stubbed(
+        :application_form,
+        first_name: 'Fred',
+        candidate: candidate,
+        application_choices: [@application_choice],
+      )
+    end
+
+    it 'has the correct subject and content' do
```

Comment:
Can we reuse the shared example similar to the other specs for consistency.

Reply:
good point, will update!